File: LearnOpencv/8/c++/main.cpp
#include <iostream>
#include <opencv2/highgui/highgui.hpp>
#include <opencv2/opencv.hpp>
using namespace std;
using namespace cv;

int main()
{
    //创建一个空图像便于画图
    Mat background=Mat::zeros(480,640,CV_8UC3);

    //画线
    line(background,Point(50,50),Point(50,150),Scalar(0,0,255),5);

    //画点
    circle(background,Point(300,50),5,Scalar(0,0,255),-1);

    //画圆
    circle(background,Point(300,200),30,Scalar(255,0,0),5);

    //画矩形
    rectangle(background,Point(500,300),Point(600,450),Scalar(0,255,0),5);
    
    //显示图像
    imshow("Background",background);

    //等待按键关闭
    waitKey(0);
    return 0;
}

File: LearnOpencv/8/python/main.py
import cv2
import numpy as np

# 创建一个空图像便于画图
background=np.zeros((480,640,3),dtype=np.uint8)

# 画线
cv2.line(background,(50,50),(50,150),(0,0,255),5)

#画点
cv2.circle(background,(300,50),5,(0,0,255),-1)

#画圆
cv2.circle(background,(300,200),30,(255,0,0),5)

#画矩形
cv2.rectangle(background,(500,300),(600,450),(0,255,0),5)

# 显示图像
cv2.imshow("Background",background)

# 等待按键关闭
cv2.waitKey(0)
